### PR TITLE
Add pingdom.Pause/Unpause actors.

### PR DESCRIPTION
**Pull request by siminm**
Handles issue #224

### CODE REVIEW

File: kingpin/actors/pingdom.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+from tornado import gen
+
+from kingpin.constants import REQUIRED
+from kingpin.actors import base
+from kingpin.actors import exceptions
+from kingpin.actors.support import api
+
+log = logging.getLogger(__name__)
+
+__author__ = 'Matt Wise <matt@nextdoor.com>'
```
Comment (diranged) [inline kingpin/actors/pingdom.py]: O really? :open_mouth:

File: kingpin/actors/pingdom.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        super(PingdomBase, self).__init__(*args, **kwargs)
+
+        rest_client = api.RestClient(
+            headers={'App-Key': TOKEN}
+        )
+        self._pingdom_client = PingdomAPI(client=rest_client)
+
+    @gen.coroutine
+    def _get_check(self):
+        resp = yield self._pingdom_client.checks().http_get()
```
Comment (diranged) [inline kingpin/actors/pingdom.py]: Quick doc string just describing the goal of this method. Is it purely for testing, does it do real useful stuff even in the real run?

**Comment by diranged** [comment]
Add integration tests... then we're good.

File: kingpin/actors/pingdom.py (hunk 1 above)
Comment (siminm) [inline kingpin/actors/pingdom.py]: lol, hah! good job, Matt!

**Comment by diranged** [comment]
:+1: 
Looks good .. have you run the integration tests?

**Comment by siminm** [comment]
Can't figure out pingdom :( Created a new user and it doesn't see any of the original "checks"

**Comment by siminm** [comment]
Integration tests pass!

**Pull request merged**